src/meta/processors/admin/Balancer.cpp

```diff
@@ -389,6 +398,39 @@ bool Balancer::balanceParts(BalanceID balanceId,
 
   auto maxPartsHost = sortedHosts.back();
   auto minPartsHost = sortedHosts.front();
+  if (innerBalance_) {
```

Comment:
What is `innerBalance_`?
What is the relationship with `update` function...

Reply:
`innerBalance_ ` is used to move parts in one zone. `update` is try to collect the modify into one place...

Reply:
so I have remove temporarily